main.go

```diff
@@ -18,15 +17,15 @@ type RequestObject struct {
 }
 
 func main() {
-	port := os.Getenv("PORT")
-	if port == "" {
-		panic("Failed to load variable $PORT from environment")
-	}
+	// port := os.Getenv("PORT")
+	// if port == "" {
+	// 	panic("Failed to load variable $PORT from environment")
+	// }
 
 	http.Handle("/", http.FileServer(http.Dir("./static")))
 	http.HandleFunc("/interpret", handleInterpret)
 
-	http.ListenAndServe(":"+port, nil)
+	http.ListenAndServe(":8080", nil)
```

Comment:
you have to leave it as it is and just export `PORT` variable when developing, we need this on heroku.